README.md

```diff
@@ -0,0 +1,5 @@
+# JavaBasics
+A playground for all the happy Java learners outside.
+
+Travis just reported the latest status of our project is 
+[![Build Status](https://travis-ci.org/mle-enso/JavaBasics.svg?branch=master)](https://travis-ci.org/mle-enso/JavaBasics)
```

Comment:
Of course we have to adapt the links to the current build status to something like https://travis-ci.org/evainga/JavaBasics as soon as you enabled Travis-support for your repo.